hack/make-rules/test-cmd-util.sh

```diff
@@ -3694,6 +3695,21 @@ __EOF__
     rm "${SAR_RESULT_FILE}"
   fi
 
+  # kubectl auth can-i
+  # kube-apiserver is started with authorization mode AlwaysAllow, so kubectl can-i always returns yes
```

Comment:
I can add some abnormal test cases when this is merged:
https://github.com/kubernetes/kubernetes/pull/46432